docs/UnixSignals/LowVoltage/index.md

```diff
@@ -0,0 +1,60 @@
+## Low Voltage  
+
+Type
+: Signal
+
+Sender
+: HMI
+
+Purpose
+: To suspend/restore SDL's services due to a low voltage event
+
+### Description  
+A 'LowVoltage' event occurs on HMI when battery voltage hits below a certain predefined threshold set by the system.  
+In case of such event, SDL operations are halted including receiving and processing of normal RPC messages from HMI. 
+HMI sends system signals of the range SIGRTMIN - SIGRTMAX for LowVoltage, WakeUp and IgnitionOff to SDL. 
+After the voltage level is restored HMI sends `WAKE_UP` signal to SDL and SDL processes all operations resumption.
+
+!!!
+MUST  
+1. Send `LOW_VOLTAGE` signal to SDL when a 'LowVoltage' event occurs
+2. Send `WAKE_UP` signal to SDL when the voltage recovers
+3. Send `IGNITION_OFF` signal when SDL processes have to be terminated, applications have to be unregistered due SDL functionality suspension during a LOW VOLTAGE event
+!!!
+
+#### SDL behavior in case of a `LOW_VOLTAGE` event:
+* SDL ignores all the requests from mobile applications without providing any kind of response.
+* SDL ignores all responses and messages from HMI except for `WAKE_UP` or `IGNITION_OFF` signals.
+* SDL stops audio/video streaming services.
+* All transports are unavailable for SDL.
+* SDL persists resumption related data stored before receiving a `LOW_VOLTAGE` signal.
+* SDL and the PoliciesManager must persist 'consumer data' (resumption-related + local PT) periodically and independently of the external events. 
+
+#### SDL resumes its regular work after receiving a "WAKE_UP" signal:
+* After receiving a `WAKE_UP` signal, all applications will be unregistered and the device disconnected.
```

Comment:
@KhrystynaDubovyk This point is missed in diagram

Reply:
@AByzhynar, thank you
fixed with  https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/109/commits/7047ffd3a19230ba5c5af90345d89c7d1e0080d9, https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/109/commits/0264c5ebfc2e9ef429e6511afb0a12773ee62840